langserver/symbol.go

```diff
@@ -59,6 +62,7 @@ func (q Query) String() string {
 	for _, token := range q.Tokens {
 		s = queryJoin(s, token)
 	}
+	s = queryJoin(s, fmt.Sprintf("CaseSensitive:%v", q.CaseSensitive))
```

Comment:
this should go before the above for loop. It would also be nice that if q.CaseSensitive is the default value, that it isn't adding to the query string.